src/context/MainContext.jsx

```diff
@@ -15,6 +15,15 @@ export const MainProvider = ({ children }) => {
     JSON.parse(localStorage.getItem("darkTheme")) || false
   );
 
+  const [isSmallText, setIsSmallText] = useState(
+    JSON.parse(localStorage.getItem("smallText")) || false
```

Comment:
Nice! But I think we should keep it enabled on mobile view. You can use `useMediaQuery` hook for that and change the default value

Reply:
I'll request PR again by tomorrow.

Reply:
`useMedaiQuery` doesn't work in `MainContext`. So I solved it in `Todo` component.